pkg/segment/writer/segstore.go

```diff
@@ -792,12 +811,18 @@ func (segstore *SegStore) initStarTreeCols() ([]string, []string) {
 		}
 
 		cest := uint32(segstore.AllSst[cname].Hll.Estimate())
-		gcMap[cname] = cest
+
+		// skip columns if card is above threshold
+		if cest > uint32(wipCardLimit*2) {
+			continue
+		}
```

Comment:
nit: this comment isn't really useful, especially if you rename `cest` to something more clear